inlong-dashboard/src/pages/Clusters/CreateModal.tsx

```diff
@@ -101,7 +121,17 @@ const Comp: React.FC<Props> = ({ id, defaultType, ...modalProps }) => {
     <Modal
       {...modalProps}
       title={id ? i18n.t('pages.Clusters.Edit') : i18n.t('pages.Clusters.Create')}
-      onOk={onOk}
+      footer={[
+        <Button onClick={modalProps.onCancel}>{i18n.t('basic.Cancel')}</Button>,
+        <Button type="primary" onClick={onOk}>
+          {i18n.t('basic.Save')}
+        </Button>,
+        (type === 'PULSAR' || type === 'KAFKA' || type === 'TUBEMQ') && (
```

Comment:
I don't think these types should be limited to testing, otherwise you need to consider this when expanding the plug-in in the future, including plug-ins for privatization deployment.